docs/platforms/elixir/configuration/logging.mdx

```diff
@@ -0,0 +1,86 @@
+
+First, configure the Sentry handler in your application's configuration:
+
+```elixir {filename:config/prod.exs}
+config :my_app, :logger, [
+  {:handler, :my_sentry_handler, Sentry.LoggerHandler, %{
+    config: %{
+      metadata: [:file, :line],
+      rate_limiting: [max_events: 10, duration: _1_second = 1_000],
+      capture_log_messages: true
+    }
+  }}
+]
+```
+
+You can use any handler identifier in place of `:my_sentry_handler`. Then, in your application's `Application.start/2` callback, add the handler to the logger:
+
+```elixir {filename:my_app/lib/application.ex}
+@impl Application
+def start(_type, _args) do
+  children = [
+    # ...
+  ]
+
+  Logger.add_handlers(:my_app)
+end
+```
+
+### Capturing Logged Messages
+
+If you also want to capture **logged messages** above a specific log level (and not just process crashes), you need to set `:capture_log_messages` and then you can use the `:level` option in the handler configuration:
+
+```elixir {filename:config/prod.exs}
+config :my_app, :logger, [
+  {:handler, :my_sentry_handler, Sentry.LoggerHandler, %{
+    config: %{capture_log_messages: true, level: :error}
+  }}
+]
+```
+
+Now, messages will be reported to Sentry even if logged with `Logger.error/1`, for example:
+
+```elixir
+Logger.error("This will be reported to Sentry")
+```
+
+## Logger Handler Configuration
+
+You can configure the `:logger` handler under the `:config` key, as shown above. The configuration options are shown below.
+
+<ConfigKey name="capture-log-messages">
+
+When this is `true`, the handler will report all logged messages to Sentry (provided they're not filtered by `:excluded_domains` and `:level`). Defaults to `false`, which will only send crash reports, which are messages with metadata that has the shape of an exit reason and a stacktrace.
+
+</ConfigKey>
+
+<ConfigKey name="level">
+
+the minimum [`Logger` level](https://hexdocs.pm/logger/Logger.html#module-levels) to send events for. Defaults to `:error`. You can lower this to `:warning`, for example, to also report `Logger.warning/1` calls.
+
+</ConfigKey>
+
+<ConfigKey name="metadata">
+
+Use this to include non-Sentry logger metadata in reports. If it's a list of keys, metadata in those keys will be added in the `:extra` context, under the `:logger_metadata` key. If set to `:all`, all metadata will be included. Defaults to `[]`.
+
+</ConfigKey>
+
+<ConfigKey name="excluded-domains">
+
+Any messages with a domain in the configured list will not be sent. Defaults to `[:cowboy]` to avoid double-reporting events from `Sentry.PlugCapture`.
+
```

Comment:
```suggestion
<ConfigKey name="excluded-domains">

This configuration option makes it so that messages with a domain in the configured list will not be sent. It's set to  `[:cowboy]` by default to avoid double-reporting events from `Sentry.PlugCapture`.

```